packages/fast-components-react-msft/src/flipper/flipper.tsx

```diff
@@ -26,7 +26,7 @@ class Flipper extends Foundation<IFlipperHandledProps & IManagedClasses<IFlipper
             <Button
                 {...this.unhandledProps()}
                 {...this.coerceButtonProps()}
-                managedClasses={this.props.managedClasses}
+                managedClasses={null}
```

Comment:
I don't think we need this

Reply:
Yes, the flipper expects managedClasses from base button, but we no longer extend the class name contract, so this overrides those managed classes.